src/MR.AspNet.Identity.EntityFramework6.Model/MR.AspNet.Identity.EntityFramework6.Model.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <Version>3.0.4</Version>
```

Comment:
There is a common.props file that I import in all csprojs that let me share lots of those values, and also reference a single version. So please check the other csprojs on how to edit this one to take advantage of common.props. You'll still need to add the `TargetFramework` element above to override that.